From: Felix Fietkau <nbd@nbd.name>
Subject: jffs2: add splice ops

Add splice_read using generic_file_splice_read.
Add splice_write using iter_file_splice_write

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---

--- a/fs/jffs2/file.c
+++ b/fs/jffs2/file.c
@@ -57,6 +57,7 @@
 	.mmap =		generic_file_readonly_mmap,
 	.fsync =	jffs2_fsync,
 	.splice_read =	generic_file_splice_read,
+	.splice_write =	iter_file_splice_write,
 };
 
 /* jffs2_file_inode_operations */
